vagrant-pxe-harvester/README.md

```diff
@@ -34,6 +34,15 @@ Quick Start
     *NOTE*: by default `harvester_vip` is `192.168.0.131`. However, it is
     configureable in `settings.yml`.
 
+Running With Single Node Air-Gapped Rancher & Harvester Node
+-----------
+1.  As above, edit `settings.yml` to make sure the configuration satisfies your
+    needs. The options are self-documented.
```

Comment:
We should add in the settings for rancher airgapped to work as a sub-bullet in the README.md. There's three of them or so and they should be documented.